### PR TITLE
Adds the ability for ERTs to use a custom shuttle

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
ERTs can now have a custom shuttle template defined on their type, which, should the "use custom shuttle" toggle be enabled while creating the ERT, will spawn them in deep space on the given shuttle instead of at Centcom. 

Applies the new system to the bounty hunter ERT

## Why It's Good For The Game
Not all ERTs are NT enough to start at Centcom all the time, and it gives more control over ERTs

## Changelog
:cl:
admin: ERTs can now have a custom shuttle that they can spawn on, on a toggle while creating said ERT.
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/ert.dm [hunk 1]
```diff
@@ -109,36 +111,73 @@
...
+				for(var/obj/effect/landmark/ert_shuttle_spawn/spawner in affected_turf)
+					spawn_turfs += get_turf(spawner)
+
+				if(!brief_spawn)
+					brief_spawn = locate(/obj/effect/landmark/ert_shuttle_brief_spawn) in affected_turf
+
+			if(!length(spawn_turfs))
+				stack_trace("ERT shuttle loaded but found no spawnpoints, placing the ERT at wherever inside the shuttle instead.")
+
+				for(var/turf/open/floor/open_turf in shuttle_turfs)
```
Comment (san7890) [inline code/modules/admin/verbs/ert.dm:161]: I get that this is unwanted behavior, but if you're going to do a fallback to "anywhere on the shuttle that might work", maybe do an `is_safe_turf` check so you don't spawn them inside an engine surrounded by walls/space.

Comment (Zonespace27) [reply code/modules/admin/verbs/ert.dm:161]: Done

**Comment by Pumpkinoe** [comment]
frickin zased

**Comment by Twaticus** [comment]
is the icon a reference or something cause he kinda just looks like a curator. maybe it should be more centcom looking?

**Comment by Zonespace27** [comment]
> is the icon a reference or something cause he kinda just looks like a curator. maybe it should be more centcom looking?

Nah it's just a clothing file I had in my "made but never used" folder that i slapped on a guy; I can probably replace it with some CC uniform

**Pull request merged**